app/views/admin/promotional_feature_items/_promotional_feature_item.html.erb

```diff
@@ -1,26 +1,82 @@
-<%= content_tag_for(:section, promotional_feature_item, class: 'well') do %>
-  <% unless promotional_feature_item.image.s300.url.blank? %>
-    <%= image_tag promotional_feature_item.image.s300.url, alt: promotional_feature_item.image_alt_text %>
-  <% end %>
+<%= render "components/summary_card_component", {
+  title: "Feature item #{promotional_feature_item.id}",
+  rows: [
+    {
 
-  <% if promotional_feature_item.youtube_video_url.present? %>
-    <p>
-      <%= link_to promotional_feature_item.youtube_video_url, promotional_feature_item.youtube_video_url %>
-    </p>
-  <% end %>
+      key: "Item title",
+      value: promotional_feature_item.title
 
-  <% if promotional_feature_item.title.present? %>
-    <p><%= link_to_unless promotional_feature_item.title_url.blank?, promotional_feature_item.title, promotional_feature_item.title_url %></p>
-  <% end %>
-  <p>
-    <%=format_with_html_line_breaks promotional_feature_item.summary %>
-  </p>
-  <p>
-    <% promotional_feature_item.links.each do |link| %>
-      &mdash; <%= link_to link.text, link.url %><br />
-    <% end %>
-  </p>
+    },
+    {
+      key: "Item title URL",
+      value: "",
+      actions: [
+        {
+          label: "View",
+          href: promotional_feature_item.title_url
+        },
+      ]
+    },
+    {
+      key: "Summary",
+      value: promotional_feature_item.summary,
+
+    },
+    if promotional_feature_item.image.s300.url.present?
```

Comment:
Hey so the easiest way to add conditional rows to an array of hashes is to use a splat operator. You can read about that more here if you're interested https://thoughtbot.com/blog/ruby-splat-operator

We can do this to pass the conditional rows in.

```
*([
      {
        key: "Image",
        value: image_tag(promotional_feature_item.image.s300.url),
        actions: [
          {
            label: "View",
            href: promotional_feature_item.image.s300.url,
          },
        ]
      },
      {
        key: "Image description",
        value: promotional_feature_item.image_alt_text,
      },
    ] if promotional_feature_item.image.s300.url.present?),
    *([
      {
        key: "YouTube video",
        value: promotional_feature_item.youtube_video_url,
        actions: [
          {
            label: "View",
            href: promotional_feature_item.youtube_video_url,
          },
        ]
      },
      {
        key: "Video description",
        value: promotional_feature_item.youtube_video_alt_text,
      },
    ] if promotional_feature_item.youtube_video_url.present?),
```